libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -272,6 +275,9 @@ class MapboxNavigation @VisibleForTesting internal constructor(
         )
     )
 
+    @ExperimentalPreviewMapboxNavigationAPI
+    private var fasterRoutesInstance: FasterRoutesTracker? = null
```

Comment:
`FasterRoutesTracker` takes coroutine scope as a parameter and stops working as soon as it cancels. `FasterRoutesTracker` was designed just to destroy `FasterTracker` earlier than `MapboxNavigation`. [I'm already thinking of removing it to avoid confusions and simplify things](https://github.com/mapbox/mapbox-navigation-android/pull/6434#discussion_r992032124).